client/my-sites/email/email-management/titan-manage-mailboxes/index.jsx

```diff
@@ -80,6 +88,75 @@ class TitanManageMailboxes extends Component {
 		} );
 	};
 
+	getResponsiveConfig = ( context, description ) => {
+		const { isMobileViewport } = this.props;
+
+		if ( isMobileViewport ) {
+			return {};
+		}
+
+		return {
+			path: this.getPath( context ),
+			description,
+		};
+	};
+
+	getTitanItems = () => {
+		const { translate, isMobileViewport } = this.props;
+		const navItemClass = isMobileViewport ? 'titan-manage-mailboxes__disable-on-mobile' : '';
+		return [
+			{
+				className: navItemClass,
+				external: true,
+				materialIcon: 'dvr',
+				text: translate( 'Configure desktop app' ),
+				...this.getResponsiveConfig(
+					TITAN_CONTROL_PANEL_CONTEXT_CONFIGURE_DESKTOP_APP,
+					translate( 'View settings required to configure third-party email apps' )
+				),
```

Comment:
It feels like the display of `description` should be completely controlled via CSS - the description always applies, the only question is whether it is visible or not on the current viewport. So I think this would be simpler to refactor as follows:
```suggestion
				description: translate( 'View settings required to configure third-party email apps' ),
				path: this.getPath( context ), // the logic for returning a mobile-friendly path can be controlled by getPath()
```

Reply:
Done